Formula/sevenzip.rb

```diff
@@ -0,0 +1,30 @@
+class Sevenzip < Formula
+  desc "7-Zip is a file archiver with a high compression ratio"
+  homepage "https://7-zip.org"
+  url "https://7-zip.org/a/7z2106-src.7z"
+  version "21.06"
+  sha256 "675eaa90de3c6a3cd69f567bba4faaea309199ca75a6ad12bac731dcdae717ac"
+  license all_of: ["LGPL-2.1-or-later", "BSD-3-Clause"]
+
+  def install
+    cd "CPP/7zip/Bundles/Alone2" do
+      mac_suffix = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch
+      mk_suffix, directory = if OS.mac?
+        ["mac_#{mac_suffix}", "m_#{mac_suffix}"]
+      else
+        ["gcc", "g"]
+      end
+
+      system "make", "-f", "../../cmpl_#{mk_suffix}.mak"
+
+      bin.install "b/#{directory}/7zz"
```

Comment:
Can we use `make install` here?

Reply:
No, we can't do that in current release, as upstream didn't ship any make rule for installation stuffs. Thanks!

Reply:
Ok, can we ask upstream to provide an `install` target for `make`? Or perhaps you might be willing to contribute that upstream.

Reply:
Sure I can submit a feature request in 7zip discussion community. Will this formula be added in the meantime? Because 7-zip has quite a long release period, we might have to take unreasonable time to wait for the release with a `install` target (if they agree to add it).

Reply:
No, but a link to the feature request in the formula will be useful so that we know to be able to switch to using `make install` when the time comes would be useful.

Reply:
Great, I've open a discussion (https://sourceforge.net/p/sevenzip/discussion/45797/thread/1d5b04f2f1/) there. Thanks!

Reply:
Great; thanks. Can you add that link in a comment in the formula?

Reply:
Done~